src/main/java/com/github/fishio/settings/Settings.java

```diff
@@ -66,7 +66,9 @@ public SimpleDoubleProperty getSliderProperty(String setting) {
 		SimpleDoubleProperty s = sliderSettings.get(setting);
 		if (s == null) {
 			logger.log(LogLevel.ERROR, "Setting '" + setting + "' not found!");
```

Comment:
@Diocruel This is quite an annoying one. We could make a private static final field for it, but that would change quite little. I would opt for keeping this PMD warning.

<review